bin/build-suggesters.sh

```diff
@@ -16,7 +16,13 @@ exec 3>&1
 
 status=0
 for suggester in propertySuggesterNoHighlight bioentitySuggester propertySuggester; do
-  HTTP_STATUS=$(curl $SOLR_AUTH -w "%{http_code}" -o >(cat >&3) -s "http://${HOST}/solr/${COLLECTION}/suggest?suggest.dictionary=${suggester}&suggest.build=true")
+  HTTP_STATUS=$(curl $SOLR_AUTH \
+  -w "%{http_code}" \
+  -o >(cat >&3) \
+  -s \
+  -v \
+  --trace-ascii debug.log \
+  "http://${HOST}/solr/${COLLECTION}/suggest?suggest.dictionary=${suggester}&suggest.build=true")
```

Comment:
```suggestion
  "http://${HOST}/solr/${COLLECTION}/suggest?suggest.dictionary=${suggester}&suggest.build=true&timeAllowed=3600000")
```
@ke4 @upendrakumbham  is it too much?